cluster/src/main/java/io/scalecube/cluster/membership/MembershipProtocolImpl.java

```diff
@@ -582,6 +586,7 @@ private Message prepareSyncDataMsg(String qualifier, String cid) {
                           metadataStore.updateMetadata(member, metadata1);
                       sink.next(MembershipEvent.createUpdated(member, metadata0, metadata1));
                     })
+                .onErrorResume(TimeoutException.class, e -> Mono.empty())
```

Comment:
`TimeoutException` is expected one and must not pollute log with _Received unexpected error_   logs.